bigchaindb/common/transaction.py

```diff
@@ -118,7 +118,7 @@ def from_dict(cls, data):
             fulfillment = Fulfillment.from_uri(data['fulfillment'])
         except ValueError:
             # TODO FOR CC: Throw an `InvalidSignature` error in this case.
-            raise InvalidSignature("Fulfillment URI couldn't been parsed")
+            raise InvalidSignature('Fulfillment URI couldn\'t been parsed')
```

Comment:
I second @ttmc: double quotes are useful to surround text that includes single quotes